code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -240,7 +240,7 @@
 //ARG Assault Rifle//
 /obj/item/gun/projectile/automatic/ar
 	name = "\improper ARG"
-	desc = "A robust assault rifle used by Nanotrasen fighting forces."
+	desc = "A robust assault rifle used by the forces of the Trans-Solar Federation."
```

Comment:
```suggestion
	desc = "A robust assault rifle used by Trans-Solar Federation forces."
```
Could do like that or other ways to avoid "the" repetition.